Include/abstract.h

```diff
@@ -397,6 +397,12 @@ PyAPI_FUNC(int) PyIter_Check(PyObject *);
    This function always succeeds. */
 PyAPI_FUNC(int) PyAIter_Check(PyObject *);
 
+/* Return 1 and set 'item' to the next item of iter on success.
```

Comment:
You should only add the function to limited C API 3.14. Add something like:

```c
#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030e0000
...
#endif
```

Reply:
Do we want to add it to the Limited API?

Reply:
Your PR adds it to the limited C API. I don't know your intent.

Reply:
I think we should add this to stable abi right away.

Reply:
You are of course right; I'll address your remark right away, Victor :)

Reply:
Addressed with 636e39be3e683663eda983a6efd9fe84b2204fab